force_proximity_ros/Proximity[] proximities
float64 pressure
uint16 r_finger_flex
uint16 l_finger_flex
